src/scss/components/_inputs.scss

```
@@ -76,15 +76,20 @@ $input-select-margin-right: 1;
   border-radius: 50%;
 }
 
-.usa-radio__input:disabled + .usa-radio__label::before,
+.usa-radio__input:disabled+.usa-radio__label::before,
```

I predict you'll get a lint error on this one.

```suggestion
.usa-radio__input:disabled + .usa-radio__label::before,
```